iceoryx_binding_c/include/iceoryx_binding_c/config.h

```diff
@@ -97,4 +97,7 @@ uint32_t iox_cfg_max_runtime_name_length();
 /// @brief the maximum size of a node name string + \0 terminator
 #define IOX_CONFIG_NODE_NAME_SIZE 101
 
+/// @brief the maximum size of a service description string identifier
+#define IOX_CONFIG_SERVICE_STRING_SIZE 100
```

Comment:
Please change it to 101

Reply:
This is not very intuitive as it is the buffer size.